src/v/cluster/partition_probe.cc

```diff
@@ -253,6 +260,17 @@ void replicated_partition_probe::setup_public_metrics(const model::ntp& ntp) {
           sm::description("Total number of records fetched"),
           labels)
           .aggregate({sm::shard_label, partition_label}),
+        sm::make_counter(
+          "request_bytes_total",
```

Comment:
should it be "bytes_fetched_from_follower_total"?

Reply:
we use label to introduce that dimension, the same way as we do for produced bytes